src/main/resources/data/proton/recipes/cobblestone_bricks/cobblestone_bricks.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "##",
+    "##"
+  ],
+  "key": {
+    "#": {
+      "item": "minecraft:cobblestone"
+    }
+  },
+  "result": {
+    "item": "proton:cobblestone_bricks",
+    "count": 4
+  },
+  "conditions": [
+    {
+      "type": "proton:flag",
+      "flag": "cobblestone_bricks"
```

Comment:
Where is this flag defined?

Reply:
I believe it depends on the module configs.

Reply:
I don't see it defined anywhere though. What mod adds `"conditions"` anyway?